### PR TITLE
FIX #18 Remove the IntegrationTest scope from the GraphQLQueryPlugin

### LINKED ISSUE #18: Remove the IntegrationTest scope from the GraphQLQueryPlugin
The validation should only be available in the `Compile` scope.
If the `IntegrationTest` is request we can add it later.

### PR DESCRIPTION
**Pull request by muuki88**

### CODE REVIEW

File: src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala [hunk 1]
```diff
@@ -25,9 +25,7 @@ object GraphQLQueryPlugin extends AutoPlugin {
   import autoImport._
   import GraphQLSchemaPlugin.autoImport._
 
-  // TODO separate these into two auto plugins
-  override def projectSettings: Seq[Setting[_]] =
-    pluginSettings(Compile) ++ pluginSettings(IntegrationTest)
+  override def projectSettings: Seq[Setting[_]] = pluginSettings(Compile)
 
   private def pluginSettings(config: Configuration): Seq[Setting[_]] =
     inConfig(config)(
```
Comment (jonas) [inline src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala]: The config can be removed from the following line below:

         graphqlQueryDirectory := (sourceDirectory in Compile).value / "graphql",

**Comment by jonas** [comment]
Refs #18

File: src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala (hunk 1 above)
Comment (muuki88) [reply src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala]: Good catch. I'm thinking about putting things into the `Test` scope as these are actually tests.

I'm also less and less sure if the `QueryPlugin` is actually necessary. You can write simple unit
tests with scalatest and sangria pretty easy as well. WDYT?

Comment (jonas) [reply src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala]: The QueryPlugin helps to check that the generated schema and associated queries. I guess that may be helpful in cases where you don’t us Scala end-to-end. In the project I used sangria in we had a scala.js client and was able to write unit tests that ran queries against the server-side and also tested the JSON serialization.

Comment (muuki88) [reply src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala]: Interesting point. Then let's keep it. I think the test scope is a better fit then, too.

File: build.sbt [hunk 2]
```diff
@@ -15,7 +15,6 @@ libraryDependencies ++= Seq(
 
 // scripted test settings
 scriptedLaunchOpts += "-Dproject.version=" + version.value
-scriptedLaunchOpts += "-Dcodegen.samples.dir=" + ((baseDirectory in ThisBuild).value / "src/test/resources")
```
Comment (muuki88) [inline build.sbt]: I isolated the test cases as it was hard to refactor things due to the inter-dependencies.
Also the scripted tests acted as unit tests as well, which is too much IMHO.

**Pull request merged**